src/index.js

```diff
@@ -165,19 +165,10 @@ const Amplitude = {
 
   /**
    * Set user id
-   * @param userID {String} required
+   * @param userID {String} or null required
    */
-  setUserId: function (userID) {
-    if(!userID) {
-      error('[setUserId] userID is required.');
-      return;
-    }
-    if(!isString(userId)) {
-      error('[setUserId] userID should be a string.');
-      return;
-    }
```

Comment:
allowing for `null` is fine with me, but can we retain the check for `null` or `string` and use the `warn()` instead of `error()`?

lodash has a null check function that could be used in addition to `isString`.

Reply:
good call 👍 on it!